src/modules/accommodations/accommodations.controller.ts

```diff
@@ -23,4 +43,36 @@ export class AccommodationsController {
       limit: searchQueryParams.limit,
     };
   }
+
+  @Post()
+  @CreateAccommodationSwaggerDecorator()
+  async createAccommodation(@Body() createDto: CreateAccommodationDto) {
+    return this.accommodationsService.createAccommodation(createDto);
```

Comment:
I recommend to return data inside data property because everyone has been doing it like thattttt
like this
```js
return {
    data: // your data here
}
```